src/app/clusters/chime-server/ChimeCluster.h

```diff
@@ -63,7 +51,7 @@ class ChimeServer : private AttributeAccessInterface, private CommandHandlerInte
      * Sets the Enabled attribute. Note, this also handles writing the new value into non-volatile storage.
      * @param Enabled The value to which the Enabled  is to be set.
      */
-    Protocols::InteractionModel::Status SetEnabled(bool Enabled);
+    Protocols::InteractionModel::Status SetEnabled(bool enabled);
```

Comment:
Since the getters and setters are public and the delegate exposes the cluster to the app, shouldn't we have some synchronization mechanism for the various Set & Get?